src/Illuminate/View/Compilers/BladeCompiler.php

```diff
@@ -619,12 +624,16 @@ protected function compileContinue($expression)
      *
      * @param  string  $expression
      * @return string
+     *
+     * @throws RuntimeException
      */
     protected function compileForelse($expression)
     {
         $empty = '$__empty_'.++$this->forelseCounter;
 
-        preg_match('/\( *(.*) +as *([^\)]*)/', $expression, $matches);
+        if (! preg_match('/\( *(.*) +as *([^\)]*)/', $expression, $matches)) {
+            throw new RuntimeException("Invalid `forelse` expression: ${expression}");
```

Comment:
Should be `Invalid 'forelse' expression: '{$expression}'.`. Note the replacement of the backticks, the changes to the interpolation, and the fullstop.